### PR TITLE
feat: add CreateJob endpoint to Job Batches API

### PR DESCRIPTION
**Pull request by vfisyuk-smartling**
CreateJob endpoint.

[JIRA Ticket](https://bt.smartling.net/browse/CON-1534)

### CODE REVIEW

File: smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobRequestPTO.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class CreateJobRequestPTO
+{
+    private String nameTemplate;
```
Comment (dimitrystd) [inline smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobRequestPTO.java:16]: Could you pls add a validation that it's not empty?

File: smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobResponsePTO.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateJobResponsePTO implements ResponseData
+{
+    private String translationJobUid;
+    private String jobName;
+    private String jobNumber;
+    private String dueDate;
```
Comment (dimitrystd) [inline smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobResponsePTO.java:18]: I don't see `dueDate` in the response. I think it makes sense to accept it too (as optional).

File: smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobRequestPTO.java (hunk 1 above)
Comment (vfisyuk-smartling) [reply smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobRequestPTO.java:16]: We have the validation in the endpoint itself. Normally we don't have any client side validations in this SDK.

File: smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobResponsePTO.java (hunk 2 above)
Comment (vfisyuk-smartling) [reply smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobResponsePTO.java:18]: These fields are exaclty the same as defined in the endpoint response, including `dueDate`.

**Pull request merged**